LSE-61.tex

```diff
@@ -1174,6 +1178,93 @@ \subsubsection{All-Sky Visualization of Data Releases}
+
+\paragraph{HiPS Linkage to Coadds}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0381}
+\textbf{ID:} DMS-REQ-0381
+
+\textbf{Specification:}
+The HiPS maps produced by the Data Management system shall provide for straightforward linkage from the HiPS data to the underlying LSST coadded images.  This SHOULD be implemented using a mechanism supported by both the LSST Science Platform and by community tools.
+
+\textbf{Discussion:}
+It is intended that this be done using the "HiPS Progenitor" mechanism introduced at the May 2018 IVOA meeting, or an evolution of it that emerges from the IVOA standardization process.
+
+\paragraph{HiPS Visualization}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0382}
+\textbf{ID:} DMS-REQ-0382
+
+\textbf{Specification:}
+The LSST Science Platform shall support the visualization of the LSST-generated HiPS image maps as well as other HiPS maps which satisfy the IVOA HiPS Recommendation, and shall provide integrated behavior, such as the overplotting of catalog entries, comparable to that provided for individual source images (e.g., PVIs and coadd tiles).
+
+\textbf{Discussion:}
+Further details will be provided in the LSST Science Platform Requirements, \citeds{LDM-554}.
+
+\paragraph{Produce MOC Maps}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0383}
+\textbf{ID:} DMS-REQ-0383
+
+\textbf{Specification:}
+Data Release Production shall include the production of Multi-Order Coverage maps for the survey data, conformant with the IVOA MOC recommendation.  A separate MOC shall be produced for each filter band for the main survey.  Additional MOCs SHOULD be produced to represent special-programs datasets and other collections of on-sky data.
+
+\textbf{Discussion:}
+It is likely to be useful to produce quite a large number of MOCs as part of releasing the data and documenting its quality.  For example, it may be useful to produce both a MOC for <em>all</em> the data from a band and for only that part of the sky for which the SRD requirements in that band have been met.  It also seems useful to produce MOCs for the deep drilling fields, etc.  It may also be useful to produce MOCs on, for instance, a nightly basis, reflecting that part of the sky for which coverage was obtained in that night.  The LSST project should engage in the work begun in 2018 on the development of standards and tools for <em>spatiotemporal</em> MOCs.
+
+The tile resolution chosen for these MOCs should be fine enough to represent the dither pattern of the survey as well as the shape of the focal plane.  Some testing should be done to determine a suitable scale.
+
+The Project should produce a technical note, during the construction era, detailing the specific plans for creation of MOCs.
+
+\paragraph{Export MOCs As FITS}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0384}
+\textbf{ID:} DMS-REQ-0384
```

Comment:
Priority 1b.  We are missing a good "use community standards" requirement at OSS level.  I think we should add one.

Reply:
To be clear, that was not a suggestion to be implemented at this time.  Perhaps there's an existing LSE-61 requirement about standards that could be used as a base for traceability?  I'll look later.